build/Dockerfile

```diff
@@ -56,3 +56,11 @@ RUN mkdir -p /etc/maven/m2 \
 USER 1000
 
 ADD build/_output/bin/kamel /usr/local/bin/kamel
+
+FROM golang:1.19 as go
```

Comment:
As you wish but it could be any version of Go, we don't really care as you already have understood

Reply:
If it has no impact I would prefer. To be honest it more a maintenance concern. It would probably make is easier when we upgrade the golang version project-wide. We already have 2 different version of temurin image, I feel that is enough :smile: .

Reply:
I see, no problem, I can change that